### PR TITLE
modal for TOS & Privacy complete

### PR DESCRIPTION
**Pull request by francotorium**
Files that have been modified are index.css which contain the styling necessary for the modals.

main.handlebars was modifed to include the TOS & Privacy policy

modal.js was also added to include the functionality of the Modals

### CODE REVIEW

File: public/css/index.css [hunk 1]
```diff
@@ -69,10 +69,6 @@ a {
 }
 @font-face {
     font-family: TrumpGothic;
-    src: url('../fonts/trump_gothic_east.ttf')
-}
-@font-face {
-    font-family: TrumpGothic;
     font-style: italic;
```
Comment (jmattiace) [inline public/css/index.css]: Any reason why we removed trump gothic east?

Comment (francotorium) [inline public/css/index.css]: That's odd. I've added imports of TrumpGothicEas italict so that it doesn't default to TrumpGothic.  It's possible it didn't find the path for italic...I've recompiled and will do another pull request.
